### DISCUSSION TITLE
MAC关闭TUN模式后无法连接到国内网站

### DISCUSSION THREAD

**Discussion by sunjianan9900** [clash-verge-rev/clash-verge-rev#1870] [Q&A]
<img width="650" alt="image" src="https://github.com/user-attachments/assets/2f2b73b5-8281-4278-8beb-e240a197ed42">
TUN模式关闭后无法连接到国内网站了，所有国内网页都无法访问，但是谷歌等可以正常访问
微信可以正常聊天，但是公众号页面打不开
<img width="860" alt="image" src="https://github.com/user-attachments/assets/1c4fd622-143a-48c6-a152-7b7a6607783c">
刷新缓存也没有用

退出clash后浏览器就直接没法上网了，chrome、edge、safari都不行

**Comment by wonfen** [comment]
请用2.0版本试试。
